design/history/README.md

```diff
@@ -0,0 +1,248 @@
+and lower the barriers to entry to get something written and sharable;
+overly strict formats and formalizations would be antithetical to this purpose.
+
+Exploration Reports typically have a *single author*.  The intent behind this
+is to encourage writing things down and forming bodies of thought which
+we can reference in the future -- without putting any burdens of
+consensus-seeking up-front, which we observe can become a barrier that makes
+sharing and recording early design processes harder.
+Correspondingly, Exploration Reports don't need to reach for full solutions,
+which means as long as the report covers reasonably interesting ground,
+it should often be possible to merge them with minimal turnaround time.
+
+You can find similar patterns used in other communities described here:
+
+- https://github.com/golang/go/wiki/ExperienceReports
+
+
+FAQ
+---
+
+### Why not use github issues?
+
+Github issues are a form of "inbox": they represent things that need active response.
+
+Github issues are a medium for discussion: that's useful, but in many ways they
+don't lend themselves to closure, nor do they make it easy to create checkpoints
+or periodic resummarizations to keep dicussions on track; and they definitely
+are not themselves suitable to being treated as artifacts for reference.
+
+There are several reasons we believe a directory of files to be superior to
+github issues for our purposes:
+
+- It's preferable to save github issues for real issues that need active engagement;
+- it's possible to link to specific headings in markdown in files in github,
+  while it is *not* possible to do this to headings within in issue bodies;
+- and it's important and desirable that conversations reach halting states;
+  we want to emphasize that "thread necros" are *bad*, and rebooting a
+  conversation is better done as a *new* conversation that to has a clear start
+  and an active resummarization of why they're present-tense.
+
+Overall: remember: We want to produce artifacts we can use as touchpoints
+in discussion, and *not* be the discussion format itself.
+That means github issues (and PRs) can still be a discussion funnel; this
+does not conflict with the idea of the design history files as useful products.
+
+### Closing github issues by migrating towards design history
+
+We sometimes use the suggestion of moving a github issue towards design history.
+
+If we want the "open" and "closed" states to have any meaning, it's critical
+that things move to the closed state.  Attempting to do design processes
+entirely in Github issues doesn't necessarily obstruct this -- but
+observationally, in practice, *it does*:
+design discussions in issues almost invariably receive protest when closed,
+even if the issue has been open (sometimes for *years*!) without comment.
+
+Yet closing issues is necessary to prevent the issue list from becoming
+a graveyard full of ghastly distracting vuvuzelas.  What to do?
```

Comment:
I kind of like the "graveyard full of ghastly distracting vuvuzelas", but I think we should stick to simpler phrasing. Perhaps simply:

> …from becoming a distraction.